pkg/controller/elasticsearch/stackmon/metricbeat.tpl.yml

```diff
@@ -17,9 +17,10 @@ metricbeat.modules:
     hosts: ["{{ .URL }}"]
     username: {{ .Username }}
     password: {{ .Password }}
-    {{- if .IsSSL }}
-    ssl.certificate_authorities: ["{{ .SSLPath }}"]
+    ssl.enabled: {{ .IsSSL }}
     ssl.verification_mode: "{{ .SSLMode }}"
```

Comment:
Nit: I wonder if we should just have the verification mode in the template directly given that the value is effectively constant.

Reply:
Yeah, that crossed my mind too.